content/blog/2022-07-05-newsletter.md

```diff
@@ -0,0 +1,167 @@
+- German: [xmpp.org](https://xmpp.org/categories/newsletter/) and [anoxinon.de](https://anoxinon.de/blog/)
+- Italian: [nicfab.it](https://www.nicfab.it/)
+- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)
+
+## XSF Announcements
+
+- The XSF membership application period for the third quarter 2022 is currently open. If you are interested in becoming a XSF member then you can [apply for membership](https://wiki.xmpp.org/web/Membership_Applications_Q3_2022).
+
+## Google Summer of Code 2022
+
+![XSF and Google Summer of Code 2022](/images/logos/GSoC_2022_Logo.png "XSF and Google Summer of Code 2022")
+
+The Google Summer of Code 2022 has lifted off and coding started a while ago! The two new contributors Patiga and PawBud who will work on open-source software projects in the XMPP environment. Read their first blog posts, too.
+  - **Patiga** works on [more flexible file transfers in Dino](https://summerofcode.withgoogle.com/programs/2022/projects/z9ixHTWZ). Mentors are **fiaxh** and **Marvin W.**.
+    - [Stateless File Sharing GSoC project](https://dino.im/blog/2022/06/stateless-file-sharing-gsoc-project/)
+  - **PawBud** works towards [adding support for A/V communication via Jingle in ConverseJS](https://summerofcode.withgoogle.com/programs/2022/projects/0nRwZN19). Mentors are **JC Brand** and **vanitasvitae**.
+    - [On-Boarding Experience with XSF (Converse)](https://xmpp.org/2022/06/on-boarding-experience-with-xsf-converse/)
+
+  Feel free to spread the word via [Mastodon](https://fosstodon.org/@xmpp/108358826402429966) or [Twitter](https://mobile.twitter.com/xmpp/status/1529199174729728000). More details in our [designated page at xmpp.org](https://xmpp.org/community/gsoc-2022/).
+
+## XSF fiscal hosting projects
+
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects. Please apply via [Open Collective](https://opencollective.com/xmpp). For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/). Current projects:
+
+- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
+- [Mellium Co-op](https://opencollective.com/mellium)
+
+## Events
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Articles
+
+JMP News: [Command UI and Better Transcriptions Coming Soon](https://blog.jmp.chat/b/june-newsletter-2022)
+
+Paul Schaub writes on PGPainless: [Reproducible Builds – Telling of a Debugging Story](https://blog.jabberhead.tk/2022/06/20/reproducible-builds-telling-of-a-debugging-story/). 
+
```

Comment:
Nicola Fabiano wrote the article [Persisting with Whatsapp use: how to persevere unknowingly. The whys of our «No»](https://notes.nicfab.it/en/posts/nowa/) on his reasons to avoid Whatsapp also considering the European Commission's actions.

Nicola Fabiano wrote the article [To be «IM apps addicted» or not to be, that is the question. Choose to be free: Snikket, the system messaging based on XMPP protocol](https://notes.nicfab.it/en/posts/snikket/) highlighting XMPP as his best solution and [Snikket](https://snikket.org) as his choice.

Reply:
https://github.com/xsf/xmpp.org/pull/1144/commits/7cb81717aa836c1aa905c3aac58e1a69397266bc